src/components/ErrorPage.jsx

```diff
@@ -0,0 +1,8 @@
+export const ErrorPage = () => {
+  return (
+    <>
+      <h2>404</h2>
+      <h3>Page Not Found</h3>
+    </>
```

Comment:
If your API responded with a custom error message, this would be a great place to display it!